packages/data-transformation/fhir-to-csv/src/parseFhir/parseFhir.py

```
@@ -0,0 +1,341 @@
+
+def is_integer(n):
+    try:
+        float(n)
+    except ValueError:
+        return False
+    else:
+        return float(n).is_integer()
+
+
+def getJsonValue(lnjsn, ln,filename = ""):
+    retVal = ""
```

Ruff (0.12.2)

Local variable `retVal` is assigned to but never used

Remove assignment to unused variable `retVal`

(F841)